### PR TITLE
[Enhancement] RHEL/7 STIG update: Add new cron content

### PR DESCRIPTION
**Pull request by redhatrises**
- Add cron.allow file ownership checks
- Add cron logging to rsyslog check

### CODE REVIEW

File: shared/xccdf/services/cron.xml [hunk 1]
```diff
@@ -64,25 +64,60 @@ accountability. Furthermore, the need to schedule tasks with <tt>at</tt> or
...
+</description>
+<ocil>
+<fileowner-check-macro file="/etc/cron.allow" owner="root"/>
+</ocil>
+<rationale>If the owner of the cron.allow file is not set to root, the possibility exists for an
+unauthorized user to view or edit sensitive information.</rationale>
+<ident prodtype="rhel7" cce="CCE-TBD" />
+<oval id="file_owner_cron_allow" />
+<ref prodtype="rhel7" nist="AC-6" disa="366" ossrg="SRG-OS-000480-GPOS-00227" stigid="021190" />
+</Rule>
```
Comment (shawndwells) [inline shared/xccdf/services/cron.xml]: Assign CCE-80378-3, add to STIG profile, through c4a87ba

File: shared/xccdf/system/logging.xml [hunk 2]
```diff
@@ -179,6 +179,32 @@ users could change the logged data, eliminating their forensic value.
...
+The output should return some similar to:
+<pre>cron.*                                                  /var/log/cron</pre>
+</ocil>
+<rationale>
+Cron logging can be used to trace the successful or unsuccessful execution
+of cron jobs. It can also be used to spot intrusions into the use of the cron
+facility by unauthorized and malicious users.
+</rationale>
+<ident prodtype="rhel7" cce="CCE-TBD" />
+<oval id="rsyslog_cron_logging" />
```
Comment (shawndwells) [inline shared/xccdf/system/logging.xml]: added CCE-80380-9, added to STIG profile via 268dbe5

**Comment by shawndwells** [comment]
@redhatrises , assign your own CCEs next time ;)

**Comment by redhatrises** [comment]
Thanks @shawndwells. I was going to add CCEs in separate PR to avoid stepping over my own toes with merge conflicts. There is a ticket to make sure that CCEs are added before the draft STIG content can be considered complete.

**Pull request merged**